client/public/0.178de8756ba26f703905.hot-update.js

```diff
@@ -0,0 +1,176 @@
+webpackHotUpdate(0,{
```

Comment:
**CRITICAL**
I dont think you meant to do this, bundle files should not be in your repo, you can put them in .gitignore file